go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	github.com/AndreasBriese/bbloom v0.0.0-20190825152654-46b345b51c96 // indirect
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/bren2010/proquint v0.0.0-20160323162903-38337c27106d
+	github.com/cheggaaa/pb v1.0.29
```

Comment:
Why'd this dependency change?

Reply:
`v1.0.29` adds Plan 9 support and also added a `go.mod` file, which declares the module as `github.com/cheggaaa/pb` instead of `gopkg.in/cheggaaa/pb.v1`. So, we need to use the new module name also.